zeitschrift-fur-medien-und-kulturforschung.csl

```diff
@@ -0,0 +1,510 @@
+        <text variable="publisher-place"/>
+      </if>
+      <else-if type="book chapter" match="any">
+        <text value="o.&#160;O."/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="main-date">
+    <choose>
+      <if is-uncertain-date="issued">
+        <text term="circa" form="short" suffix=" "/>
+      </if>
+    </choose>
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="day" form="numeric" suffix="."/>
+          <date-part name="month" form="numeric" suffix="."/>
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="original-date">
+    <choose>
+      <if is-uncertain-date="original-date">
+        <text term="circa" form="short" suffix=" "/>
+      </if>
+    </choose>
+    <choose>
+      <if type="article-journal">
+        <date variable="original-date">
+          <date-part name="month" form="short" suffix=" "/>
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else-if type="book chapter" match="any">
+        <date variable="original-date">
+          <date-part name="year"/>
+        </date>
+      </else-if>
+      <else>
+        <date variable="original-date">
+          <date-part name="day" form="numeric" suffix="."/>
+          <date-part name="month" form="numeric" suffix="."/>
+          <date-part name="year"/>
+        </date>
+      </else>
+    </choose>
+  </macro>
+  <macro name="accessed-date">
+    <date variable="accessed">
+      <date-part name="day" form="numeric" suffix="."/>
+      <date-part name="month" form="numeric" suffix="."/>
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="extra-info">
+    <text prefix=", " variable="dimensions"/>
+    <text prefix=", " variable="references"/>
+    <choose>
+      <if variable="URL">
+        <group prefix=", " delimiter=" ">
+          <text prefix="unter: " variable="URL"/>
+          <text prefix="(" macro="accessed-date" suffix=")"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="locator-following">
+    <group prefix=", " delimiter=".&#160;">
+      <label variable="locator" form="short"/>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <macro name="locator-first">
+    <choose>
+      <if type="article-journal chapter" match="any">
+	<group prefix=", hier " delimiter=".&#160;">
+	  <label variable="locator" form="short"/>
+	  <text variable="locator"/>
+	</group>
+      </if>
+      <else>
+	<group prefix=", " delimiter=".&#160;">
+	  <label variable="locator" form="short"/>
+	  <text variable="locator"/>
+	</group>
+      </else>
+    </choose>
+  </macro>
+  <citation delimiter-precedes-last="never" delimiter-precedes-et-al="never" and="text" et-al-subsequent-min="4" et-al-subsequent-use-first="1" disambiguate-add-names="true" disambiguate-add-givenname="true">
```

Comment:
Just to make sure I understand the goal here correctly: the idea is to not use et al ever on first citation? That seems risky? Would they really want 12 authors in a footnote?